whoami/whoami.go

```diff
@@ -61,6 +61,17 @@ var (
 	//fatal = log.New(os.Stderr, "", log.Lshortfile)
 )
 
+func lookupUserName() string {
```

Comment:
The way I saw it, it was fast way to test. I can take this approach:

```go
cmd := exec.Command("./whoami")
```

But that would assume we have the an executable there, we did in the `basename_test.go` too, so if you don't mind that, me neither. Let me know.